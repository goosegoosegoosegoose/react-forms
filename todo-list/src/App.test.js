import { render } from '@testing-library/react';
import App from './App';

test('smoke test', () => {
  render(<App />);
});

it("snapshot test", () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
})
