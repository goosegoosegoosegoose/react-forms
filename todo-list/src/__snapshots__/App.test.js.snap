// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<DocumentFragment>
  <div>
    <form>
      <label
        for="todo"
      >
        New Todo: 
      </label>
      <input
        id="todo"
        name="todo"
        placeholder="todo"
        type="text"
        value=""
      />
      <button>
        Add Todo
      </button>
    </form>
    <h3>
      Todo List
    </h3>
  </div>
</DocumentFragment>
`;
